<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="m_centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QCheckBox" name="m_toggleInstantCheckBox">
      <property name="text">
       <string>Instant Changes Mode</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>500</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Parameters">
       <attribute name="title">
        <string>Parameters</string>
       </attribute>
       <widget class="QSpinBox" name="m_angleBox">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>130</y>
          <width>48</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="m_scaleBox">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>160</y>
          <width>68</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="m_iterationsBox">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>190</y>
          <width>48</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="m_axiomBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>250</y>
          <width>141</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="m_constantsBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>280</y>
          <width>141</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="m_rule1Box">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>320</y>
          <width>141</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="m_rule2Box">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>350</y>
          <width>141</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="m_rule3Box">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>380</y>
          <width>141</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="m_rule4Box">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>410</y>
          <width>141</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="m_rule5Box">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>440</y>
          <width>141</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QSlider" name="m_radiusSlider">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>84</width>
          <height>15</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSpinBox" name="m_detailLvlBox">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>70</y>
          <width>48</width>
          <height>25</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>60</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Radius:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Subdivisions:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>60</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Angle:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>60</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Step:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>60</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Iterations:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Axiom:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>60</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Consts:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>60</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Rule1:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>350</y>
          <width>60</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Rule2:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>380</y>
          <width>60</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Rule3:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>410</y>
          <width>60</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Rule4:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>440</y>
          <width>60</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Rule5:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Exports">
       <attribute name="title">
        <string>Exports</string>
       </attribute>
       <widget class="QPushButton" name="m_expLsysButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>120</y>
          <width>131</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Export Results </string>
        </property>
       </widget>
       <widget class="QPushButton" name="m_expParamButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>200</y>
          <width>131</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Export LSystem </string>
        </property>
       </widget>
       <widget class="QPushButton" name="m_exportObjButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>280</y>
          <width>131</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Export .obj</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="m_filePathButton">
      <property name="text">
       <string>Build LSys From File</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="m_buildButton">
      <property name="text">
       <string>Build LSys From UI</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="m_wireframeCheckBox">
      <property name="text">
       <string>Wireframe</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="m_meshInfoCheckBox">
      <property name="text">
       <string>Vertex Count</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="m_menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>857</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="m_mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="m_statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
